### PR TITLE
MOTECH-2792: Jasmine tests for auth

### PR DESCRIPTION
**Pull request by palbecki**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by jredlarski** [comment]
test this please

**Comment by jredlarski** [comment]
test this please

File: src/auth/auth.directive.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+describe("Auth Directive", function () {
+
+    var $scope;
```
Comment (jredlarski) [inline src/auth/auth.directive.spec.js]: @palbecki remove $

File: src/auth/auth.directive.spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    var AuthCtrl;
+    var AuthServiceMock;
```
Comment (jredlarski) [inline src/auth/auth.directive.spec.js]: I can't see these two variables being used anywhere, remove them if they are unnecessary

File: src/auth/login/login.directive.spec.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+describe('Login Directive', function() {
+    var $scope, $compile, element;
```
Comment (jredlarski) [inline src/auth/login/login.directive.spec.js]: @palbecki Remove $ from variables

File: src/auth/reset_password/reset-password.directive.spec.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+describe('Reset password Directive', function() {
+    var element, $scope;
```
Comment (jredlarski) [inline src/auth/reset_password/reset-password.directive.spec.js]: @palbecki Remove $ from $scope variable

**Comment by jredlarski** [comment]
@palbecki Additionally please resolve conflicts

**Comment by jredlarski** [comment]
test this please

**Pull request merged**